screenwriter/screenwriter.go

```diff
@@ -0,0 +1,60 @@
+package screenwriter
+
+import (
+	"fmt"
+	"io"
+	"strings"
+)
+
+type ScreenWriter struct {
+	writer   io.StringWriter
+	previous string
```

Comment:
`previous` what? Is this the previous value of the screen? Is there a clearer name to use?

Reply:
I've called it `previousMessage` and added the word "message" to a few other places.